function throwError() {
    throw new Error("First error!");
}

module.exports = throwError;